AdyenCard/Utilities/CardType/CardTypeProvider.swift

```diff
@@ -32,45 +38,50 @@ internal final class CardTypeProvider: Component {
     /// - Parameters:
     ///   - bin: Card's BIN number. If longer than `minBinLength` - calls API, otherwise check local Regex,
     ///   - caller:  Callback to notify about results.
```

Comment:
maybe change the docs as well: `caller -> completion`

Reply:
It moved to line 52. `didChange (detectedCards:)` is not looking nicely under throttling, so I've moved it out.

Reply:
ups :D